parm/forecast/regional/nems.configure.atm_ocn.tmp

```diff
@@ -49,3 +46,35 @@ OCN_attributes::
   end_min = 0
   end_sec = 0
 ::
+
+DRIVER_attributes::
+  start_type = startup
+::
+
+ALLCOMP_attributes::
+  mediator_read_restart = false
+# ScalarFieldCount = 3
```

Comment:
This part needs to be enabled once we bring cpl_scalars to make available mediator history and restart files in 2d for FV3. It is already done for HYCOM.

Reply:
@uturuncoglu 
I think you mean the other template,
parm/forecast/regional/nems.configure.atm_ocn_cmeps.tmp 

Also note that at that point we will have to change ocn_export_fields and ocn_import_fields. These settings in nems.configure build the HYCOM import and export field lists at runtime.